adoc/deployment-azure.adoc

```diff
@@ -0,0 +1,217 @@
+== Deployment on Microsoft Azure
+
+Deployment on Microsoft Azure is currently a tech preview.
+
+.Preparation Required
+[NOTE]
+====
+You must have completed <<deployment.preparations>> to proceed.
+====
+
+You will use {tf} to deploy the whole infrastructure described in
+<<architecture-azure>>. Then you will use the `skuba` tool to bootstrap the
+{kube} cluster on top of it.
+
+
+[[architecture-azure]]
+=== AZURE Deployment
+
+The AZURE deployment created by our {tf} template files leads to the
+creation of the infrastructure described in the next paragraphs.
+
+==== Setup service principal for terraform login credential
```

Comment:
This should be refreshed according to Chinya doc PR. (Chinya should be able to drive the update)